recipes/correct-ts-specifiers/src/map-imports.test.ts

```diff
@@ -58,11 +64,12 @@ describe('Map Imports', { concurrency: true }, () => {
 		assert.equal(output.replacement, undefined);
 		assert.notEqual(output.isType, true);
 
-		const { 0: sourcePath, 2: msg } = mock__log.calls[0].arguments;
+		assert.equal(mock__log.calls.length, 1, 'Expected exactly one error log call');
+		const [errorMessage] = mock__log.calls[0].arguments as [string];
```

Comment:
nit

```suggestion
		const errorMessage = mock__log.calls[0].arguments[0] as string;
```

It should know that is a string though 🤔